server/public/model/client4.go

```diff
@@ -9317,3 +9317,14 @@ func (c *Client4) GetFilteredUsersStats(ctx context.Context, options *UserCountO
 	}
 	return &stats, BuildResponse(r), nil
 }
+
+// DeleteSidebarCategory deletes a sidebar category for a user in a team.
+func (c *Client4) DeleteSidebarCategory(ctx context.Context, userId string, teamId string, categoryId string) (*Response, error) {
```

Comment:
Can you rename this to `DeleteSidebarCategoryForTeamForUser` to match the name of the other sidebar methods, and can you move it to be defined in the same place in the file as the rest of those?